### PR TITLE
#165413063 signup and register a user

### PR DESCRIPTION
**Pull request by ericnyirimana**
Description
=======

This PR is for registering user and activate a user account for accessing the authors heaven

Type of change
=======
N/A

How has it been tested
=======
N/A

Checklist:
=======
- Add components for sending an email activating a user.
- Add necessary HTML and scss to the page

Pivotal tracker story ID
=======
[#165413063](https://www.pivotaltracker.com/n/projects/2326984/stories/165413063)

Screenshots
=======
Screen Shot 2019-07-03 at 11 55 35

### CODE REVIEW

File: src/actions/verification.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  import http from "../helpers/httpServices";
+  export const verification = (email, token) => dispatch => {
+    dispatch({
+      type: ACCOUNT_VERIFICATION,
+      payload: http.get(`/api/v1/users/verification?email=${email}&token=${token}`)
+    }).catch(() => {
+      Helper.setAlertError("Account already verified");
+    });
+  };
+
```
Comment (AJAkimana) [inline src/actions/verification.js]: Add a new line at the end of this file

File: src/components/common/input.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Input.propTypes = {
+	name: PropTypes.string,
+	type: PropTypes.string,
+	value: PropTypes.string,
+	onChange: PropTypes.func,
+	placeholder: PropTypes.string,
+	id: PropTypes.string,
+	validateMessage: PropTypes.object
+};
+export default Input;
```
Comment (AJAkimana) [inline src/components/common/input.jsx]: New line

**Review by AJAkimana — COMMENTED** [review]
Good work @ericnyirimana, add new lines at the end of files

**Review by AJAkimana — APPROVED** [review]
Good work

**Review by Kabalisa — APPROVED** [review]

File: cypress/integration/examples/signup.spec.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      cy.get("#email")
+        .type("email@account.com")
+        .should("have.value", "email@account.com");
+      cy.get("#password")
+        .type("Password@123!")
+        .should("have.value", "Password@123!");
+      cy.get(".input").click();
+      cy.get(".Toastify").its('length').should("be.gte", 0);
+    });
+  });
```
Comment (macphilips) [inline cypress/integration/examples/signup.spec.js]: new line

File: src/utils/errorHandler.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+     if (!action.meta || !action.meta.localError) {
+      return next(action).catch(error => {
+        Notify.setAlertError(error.message);
+      });
+    }
+
+     return next(action);
+  };
+};
+export default errorHandler;
```
Comment (macphilips) [inline src/utils/errorHandler.js]: add new line

File: webpack/base.config.js [hunk 5]
```diff
@@ -21,14 +21,14 @@ module.exports = {
     ]
   },
   resolve: {
-    extensions: ['.js', '.jsx', '.es6']
+    extensions: [".js", ".jsx", ".es6"]
```
Comment (macphilips) [inline webpack/base.config.js]: use single-quote over double

**Pull request merged**